src/sas/qtgui/Plotting/LinearFit.py

```diff
@@ -224,13 +224,13 @@ def fit(self, event):
             else:
                 err = ''
 
-        self.txtGuiner_2.setText(value)
-        self.txtGuiner2_Err.setText(err)
+        self.txtGuinier_2.setText(value)
+        self.txtGuinier2_Err.setText(err)
 
         value = formatNumber(rg * self.floatInvTransform(self.xminFit))
-        self.txtGuiner_3.setText(value)
+        self.txtGuinier_4.setText(value)
         value = formatNumber(rg * self.floatInvTransform(self.xmaxFit))
-        self.txtGuiner_4.setText(value)
+        self.txtGuinier_3.setText(value)
```

Comment:
so this is the actual fix, all the rest is typos being corrected in source, which might not be that important.
Assuming unit tests pass after this (I would expect some elements are being tested by property name), this is ready for merge.

Reply:
so are you OK with merging it into 5.0.4 RC3 (assuming tests pass)? or are you concerned that it should remain only in main at this point?

Reply:
Running the tests (locally) I don't see any failures related to the Linear Fit. For some reason, no unit tests touch the Guinier text boxes.

Reply:
Then it's moot. I'm OK with it being merged.